src/Internal/Test/E2E/Runner.purs

```diff
@@ -478,21 +539,38 @@ readExtensionParams extensionName mbCliOptions = do
     { crxFile: a.crxFile <|> b.crxFile
     , password: a.password <|> b.password
     , extensionId: a.extensionId <|> b.extensionId
+    , crxUrl: a.crxUrl <|> b.crxUrl
     }
 
-  toExtensionParams :: ExtensionOptions -> Effect (Maybe ExtensionParams)
-  toExtensionParams { crxFile, password, extensionId } =
+  toExtensionParams :: ExtensionOptions -> Aff (Maybe ExtensionParams)
+  toExtensionParams { crxFile, password, extensionId, crxUrl } =
     case crxFile, password, extensionId of
       Nothing, Nothing, Nothing -> pure Nothing
-      Just crx, Just pwd, Just extId -> pure $ Just
-        { crx, password: pwd, extensionId: extId }
-      _, _, _ -> throw $ "Please ensure that either none or all of"
+      Just crx, Just pwd, Just extId -> do
+        doesExist <- exists crx
+        unless doesExist $ do
+          liftEffect $ log $ crx <> " does not exist."
```

Comment:
Do not log this line here. The test suite should not be too verbose. It's a normal part of the workflow, no need to tell the user anything.